CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Cache all calls to `os.getenv` via custom module [PR #231](https://github.com/3scale/apicast/pull/231)
 
 ### Fixed
+- [OAuth] Return correct state value back to client
```

Comment:
sorry, there needs to be a space after, you can see the error in the GitHub visual diff 
I guess we should keep them with spaces already in the template.